### DISCUSSION TITLE
Modification landing page

### DISCUSSION THREAD

**Discussion by valeriemerle** [betagouv/ma-cantine#1282] [Idées]
A changer dans la partie "mesure de la loi" (sur les cartes) sur https://ma-cantine.beta.gouv.fr/mesures-phares/diversification-des-menus :
"Expérimentation du menu végétarien dans les collectivités volontaires" à remplacer par "Expérimentation de l'option végétarienne quotidienne pour les collectivités volontaires" plus cohérent avec ce que l'on trouve dans le volet "diagnostic"

**Comment by valeriemerle** [comment]
Autre modif dans les mesures phares sur la mesure plastique : rajouter une ligne sur une nouvelle obligation dans la partie "Les contenants alimentaires en plastique sont interdits" :
"obligation de proposer un contenant ré utilisable ou composé de matière recyclable pour la vente à emporter"

**Comment by hfroot** [comment]
Ça me semble très clair. J'ai créé un issu : https://github.com/betagouv/ma-cantine/issues/1287
